CMakeLists.txt

```diff
@@ -27,41 +33,96 @@ option(
     ${PROJECT_IS_TOP_LEVEL}
 )
 
+set(CPACK_GENERATOR TGZ)
+
+# includes
 include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+include(CPack)
 
 add_library(beman.inplace_vector INTERFACE)
 # [CMAKE.LIBRARY_ALIAS]
 add_library(beman::inplace_vector ALIAS beman.inplace_vector)
```

Comment:
The `library` (export target) must named `inplace_vector` only!

Within beman project, this should be unique and with the this exported **ALIAS**, it is right.

Reply:
What change are you suggesting here specifically??? This comment is very confusing

Reply:
### the beman name conventions prevent using and installed `cmake config package`!

Reply:
What do you mean it prevents CMake config? Can you provide more details